infrastructure/display-api-service/docker-entrypoint.sh

```diff
@@ -2,6 +2,14 @@
 
 set -eux
 
+## Start the PHP FPM process.
+echo "Starting PHP 8.1 FPM"
+
+# first arg is `-f` or `--some-option`
+if [ "${1#-}" != "$1" ]; then
+  set -- php-fpm "$@"
```

Comment:
If you run PHP here it will block the reset of the execution of this script. Should be moved to just to the linjes before the last line.

Reply:
Moved